src/flask_cognito_lib/decorators.py

```diff
@@ -21,9 +21,8 @@
     secure_random,
 )
 
-cfg = Config()
```

Comment:
Do we need to change anything in the `__init__` method of `CognitoAuth` in `src/flask_cognito_lib/plugin.py`? Wondering there if we should support an optional `cfg: Optional[Config] = None` argument that can be overridden, or if None, uses the default Config object

Reply:
The way I did it in my application is following

```
    class ConfigCustom(Config):
    @property
    def redirect_url(self) -> str:
        return url_for(endpoint='auth.cognito', _external=True)

    @property
    def logout_redirect(self) -> str:
        return url_for(endpoint='auth.cognito_post_logout', _external=True)


class CognitoAuthCustom(CognitoAuth):
    """
    Custom CognitoAuth class to override
    """
    def init_app(self, app: Flask) -> None:
        self.cfg = ConfigCustom()
        app.extensions[self.cfg.APP_EXTENSION_KEY] = self
```

A bit hacky but works. 
I'll consider your suggestion with extending the available parameters.